src/components/addBook.js

```diff
@@ -19,14 +19,15 @@ const AddBook = () => {
     setAuthor(author);
   };
 
-  const handleSubmit = (event) => {
+  const handleSubmit = async (event) => {
     event.preventDefault();
     const category = 'Code';
-    dispatch(addBook({
-      title, author, category, item_id: `Item${length + 1}`,
+    await dispatch(addBook({
+      title, author, category, itemId: `Item${length + 1}`,
     }));
     setAuthor('');
     setTitle('');
+    dispatch(fetchBooks());
```

Comment:
**OPTIONAL** - I suggest calling this function in your `addBook` action. Please fix this 🙏. You can check this [article](https://stackoverflow.com/questions/68871562/dispatch-action-on-the-createasyncthunk) for how to dispatch action inside a `createAsyncThunk` function 👍 👍